docs/version-overrides.md

```diff
@@ -2,55 +2,44 @@
 This document lists the version overrides for vulnerable (nested) dependencies and the reason.
 
 ## trim
-| Override:           | >=0.0.3 |
-|:--------------------| :-------------|
-|                     | |
-| **high**        | Regular Expression Denial of Service in trim  |
-| Package             | trim |
-| Vulnerable versions | <0.0.3 |
-| Patched versions    | >=0.0.3 |
-| More info           | https://github.com/advisories/GHSA-w5p7-h5w8-2hfq  |
+┌─────────────────────┬───────────────────────────────────────────────────┐
```

Comment:
1. Did you check whether the overrides for `trim`, `trim-newlines` and `glob-parent` are still required, even after update `storybook`? 
2. This looks now like this in markdown:
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/66327622/217959648-c0e0c452-eaa4-44f4-a701-a982b9faa763.png">

Reply:
> Did you check whether the overrides for trim, trim-newlines and glob-parent are still required, even after update storybook?

Yes they are still needed unfortunately. 

> This looks now like this in markdown:

Reverted in https://github.com/SAP/open-ux-tools/pull/891/commits/451a016a0d053b28afc0ca63f172884a71ef2bc1

Reply:
ok, thanks @devinea